docs/source/guides/development/getting_started.rst

```diff
@@ -151,8 +162,8 @@ the role of a ``Teacher``\ , or "seednode":
 
    from nucypher.characters.lawful import Ursula
 
-   seed_uri = "gemini.nucypher.network:9151"
-   seed_uri2 = "104.248.215.144:9151"
+   seed_uri = "111.111.111.111:9151"
+   seed_uri2 = "222.222.222.222:9151"
```

Comment:
<SEEDNODE_URI> like other examples?

Reply:
I think - just pick a DNS name we expect to work. `mainnet.nucypher.network` or similar.

Reply:
I hesitate to have us be pigeon-holed into a name, or have to change the docs later. I'll use `<SEEDNODE URI>` and `<OTHER SEEDNODE URI>`

Reply:
✔️